src/transformers/models/detr/feature_extraction_detr.py

```diff
@@ -939,3 +1005,352 @@ def get_ids_area(masks, scores, dedup=False):
                 predictions = {"png_string": out.getvalue(), "segments_info": segments_info}
             preds.append(predictions)
         return preds
+
+    def post_process_object_detection(
+        self, outputs, object_detection_threshold: float = 0.5, target_sizes: Union[TensorType, List[Tuple]] = None
+    ):
+        """
+        Converts the output of [`DetrForObjectDetection`] into the format expected by the COCO api. Only supports
+        PyTorch.
+
+        Args:
+            outputs ([`DetrObjectDetectionOutput`]):
+                Raw outputs of the model.
+            object_detection_threshold (`float`, *optional*):
```

Comment:
As mentioned by Niels on the internal doc, just `threshold` is enough here. The method is named `post_process_object_detection` so it's obvious we are doing this.